.github/workflows/automatic-release.yml

```
@@ -38,8 +38,20 @@ jobs:
             security.ubuntu.com:80
             uploads.github.com:443
 
+      - name: Derive target branch from milestone title
+        id: vars
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:3:29: PATCH appears unused. Verify use (or export if used externally)

(shellcheck)